utils/psh.md

```diff
@@ -1,3 +1,31 @@
 # Phoenix Shell (psh)
 
-Poenix Shell is a simple command interpreter alowing to use basic Phoenix-RTOS functions.
\ No newline at end of file
+Phoenix Shell is a simple command interpreter alowing to use basic Phoenix-RTOS functions.
```

Comment:
Maybe it marketing, but it may fit better:
```suggestion
Phoenix Shell is a compact program that enables you to command the Pheonix RTOS from a command line.
```

Reply:
I think that second is more slick and will be added as you wrote it.